Api/src/controllers/officeController.js

```diff
@@ -0,0 +1,73 @@
+import {
+  v4 as uuidv4
+} from 'uuid';
+import express from 'express';
+
+let offices = [{
+  id: uuidv4(),
+  "type": "Federal Lawmaker",
+  "name": "Senate"
+}];
+
+import dataFile from "../data.json";
+
+export const getOffices = async (req, res, next) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'application/json');
+  res.send(JSON.parse(JSON.stringify(dataFile)));
+  next();
+};
+
+export const createOffice = (req, res, next) => {
+  let office = req.body;
+
+  // For creating unique office IDs
+  const officeId = uuidv4();
+
+  const officeWithId = {
+    ...office,
+    id: officeId
+  };
+
+  offices.push(officeWithId);
+
+  res.send(`Office with the post ${office.post} was added to the database`);
```

Comment:
Is this what the spec document expects you to return after creating the office? Please check and update the endpoints according to the spec before next review

Reply:
Done.